cryocare/internals/CryoCARE.py

```diff
@@ -254,6 +254,7 @@ def predict_tiled(keras_model, even, odd, output, s_src_out, s_dst_out, mean, st
             pred = pred[src]
         if pbar is not None:
             pbar.update()
+        output[:] = pred[:]
```

Comment:
I think this change has to be reversed and we need a different fix for the empty prediction that you encountered.